lib/elixir/lib/process.ex

```diff
@@ -233,6 +233,12 @@ defmodule Process do
   the current process will sleep forever, and not
   consume or reply to messages.
 
+  > #### Note {: .info }
+  >
+  > Before Elixir 18, `sleep/1` did not accept integer timeout values greater
+  > than `16#ffffffff`, that is, `2^32-1`. Since Elixir 18, arbitrarily high integer
+  > values are accepted.
```

Comment:
```suggestion
  > #### Sleeping limit {: .info }
  >
  > Before Elixir v1.18, `sleep/1` did not accept integer timeout values greater
  > than `16#ffffffff`, that is, `2^32-1`. Since Elixir v1.18, arbitrarily high integer
  > values are accepted.
```